config/config.php

```diff
@@ -339,6 +348,14 @@
     )
 );
 
+/*$tidbit_relationships['ProductCategories'] = array(
+    'ProductCategories' => array(
+        'you' => 'parent_id',
+        'self' => 'id',
+        'table' => 'product_categories'
+    ),
+);*/
+
```

Comment:
@jbarlowsugar - Can we delete this section instead of commenting and leaving it in?